client-rpc/src/rpc/staking_rpc.rs

```diff
@@ -479,9 +468,6 @@ fn get_node_metadata(
         .err_kind(ErrorKind::InvalidInput, || "invalid base64")
         .map_err(to_rpc_error)?;
 
-    #[cfg(not(feature = "mock-enclave"))]
-    verify_keypackage(&keypackage).map_err(to_rpc_error)?;
-
```

Comment:
Why remove?

Reply:
yes, to remove dependency of mls,ra-client in client-core

Reply:
But we should verify this in client.

Reply:
it's supported in dev-utils
dev-utils keypackage verify